src/main/java/org/cbioportal/persistence/mybatis/util/PaginationCalculator.java

```diff
@@ -0,0 +1,23 @@
+package org.cbioportal.persistence.mybatis.util;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
This doesn't needs to be a component. All these methods can be turned into static methods.

Reply:
@JREastonMarks  I agree. However, I merely changed and renamed the exisiting OffsetCalculator class. I would propose to change this in a followup pull request.